runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java

```diff
@@ -86,6 +86,50 @@ public static void subscribe(@Nonnull LogReplicationListener clientListener, @No
         log.info("Client subscription at timestamp {} successful.", subscriptionTimestamp);
     }
 
+    public static void subscribeRqListener(@Nonnull LogReplicationRoutingQueueListener clientListener, @Nonnull String namespace,
```

Comment:
Wont this method get invoked from CorfuStore?  Changes in CorfuStore will be required

Reply:
This is invoked from the subscription logic. Currently in the listener and the test. Yes, We need to expose this via corfuStore as well. I can add this in the subsequent change.